resources/logging/Chart.yaml

```diff
@@ -1,12 +1,13 @@
----
```

Comment:
We are talking here about the Kyma logging helm chart which is bundling Loki. So the name should be "logging" and all the author, info, sources, icon data should be kyma related, not Loki.
I think the kubeversion field should be removed as well

Reply:
naming adjusted